torch/distributed/elastic/rendezvous/c10d_rendezvous_backend.py

```diff
@@ -4,16 +4,17 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
 
+import tempfile
```

Comment:
Nit: The import statements are not alphabetically sorted. We use [black](https://github.com/psf/black) for consistent formatting of our Python files. Make sure you run it before your PRs:

```
$ black --line-length 100 <path_to_file>
```